dotCMS/src/integration-test/java/com/dotmarketing/portlets/categories/business/CategoryFactoryTest.java

```diff
@@ -0,0 +1,273 @@
+package com.dotmarketing.portlets.categories.business;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotmarketing.business.FactoryLocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.categories.model.Category;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/***
+ * Category Factory Test
+ */
+public class CategoryFactoryTest extends IntegrationTestBase {
```

Comment:
Apparently, a DotSecurityException is not thrown in many tests. I suggest to remove it from the javadocs

Reply:
done